.travis.yml

```diff
@@ -3,6 +3,7 @@ php:
   - '5.4'
   - '5.5'
   - '5.6'
+  - '7.0'
```

Comment:
Hi @chaveiro, is there are reason you have added PHP7 checks? It causes Travis CI tests to fail https://travis-ci.org/emoncms/emoncms/jobs/254672803